notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>Because you already show the CMDs for Leo I separately, why don't you try to plot g-r and r-i CMDs side by side in a single plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>